cdap-cli/src/main/java/co/cask/cdap/cli/command/CreateDatasetInstanceCommand.java

```diff
@@ -52,21 +53,32 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     String datasetName = arguments.get(ArgumentName.NEW_DATASET.toString());
     String datasetPropertiesString = arguments.getOptional(ArgumentName.DATASET_PROPERTIES.toString(), "");
     String datasetDescription = arguments.getOptional(ArgumentName.DATASET_DESCRIPTON.toString(), null);
+    String datasetOwner = arguments.getOptional(ArgumentName.OWNER_PRINCIPAL.toString(), null);
```

Comment:
We can change it to "owner". That sounds better.
I am not very sure as what is the policy for CLI backward compatibility. But if you mean to say a user have a script which runs CLI command to create dataset instance then yes that will break. Since now we expect the user to say properties k=v, description theDescription etc.